conda_build/exceptions.py

```diff
@@ -126,3 +128,8 @@ def __init__(self, error, *args):
         self.error = error
         self.msg = "runpaths check failed \n%s" % (error)
         super().__init__(self.msg)
+
+
+class CondaBuildUserError(CondaError):
```

Comment:
Is this like ValueError?

Reply:
for our exception handling purposes it's more like `SystemExit`, so a clean exit printing the error message without the error reporting/stacktrace